hazelcast/src/test/java/com/hazelcast/internal/management/ThreadDumpRequestTest.java

```diff
@@ -0,0 +1,62 @@
+package com.hazelcast.internal.management;
+
+import com.eclipsesource.json.JsonObject;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.internal.management.request.ThreadDumpRequest;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.TestHazelcastInstanceFactory;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class ThreadDumpRequestTest extends HazelcastTestSupport {
+
+    private TestHazelcastInstanceFactory factory;
+
+    private ManagementCenterService managementCenterService;
+
+    @Before
+    public void setUp() {
+        factory = createHazelcastInstanceFactory(2);
+        HazelcastInstance hz = factory.newHazelcastInstance();
+        factory.newHazelcastInstance();
+
+        managementCenterService = getNode(hz).getManagementCenterService();
+    }
+
+    @After
```

Comment:
We should not repeat this at every test since it causes noise. Normally the HazelcastTestSupport.createHazelcastInstance usage takes care of closing anything that needs to be closed.